src/app/pages/resources/download/download.jsx

```diff
@@ -1,16 +1,20 @@
 import React, { useContext, useEffect, useState } from 'react';
 import {
+  Box,
   Button,
   Card,
   CardContent,
   CardHeader,
   CardActions,
+  Link,
   Divider,
   MenuItem,
   Typography,
   makeStyles,
 } from '@material-ui/core';
 import GetAppIcon from '@material-ui/icons/GetApp';
+import ReportProblemOutlinedIcon from '@material-ui/icons/ReportProblemOutlined';
+import CodeIcon from '@material-ui/icons/Code';
```

Comment:
We usually alphabetize these
```suggestion
import CodeIcon from '@material-ui/icons/Code';
import GetAppIcon from '@material-ui/icons/GetApp';
import ReportProblemOutlinedIcon from '@material-ui/icons/ReportProblemOutlined';
```